Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py

```diff
@@ -18,15 +18,50 @@
 BLOCKED_MESSAGES = "Blocked Messages"
 DELIVERED_MESSAGES = "Delivered Messages"
 
-DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.000Z"
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+""" Helper functions """
+
+
+def get_now():
+    """ A wrapper function of datetime.now
+    helps handle tests
+
+    Returns:
+        datetime: time right now
+    """
+    return datetime.now()
+
+
+def get_fetch_times(last_fetch):
+    """ Get list of every hour since last_fetch
+    Args:
+        last_fetch (datetime or str): last_fetch time
+
+    Returns:
+        List[str]: list of str represents every hour since last_fetch
+    """
+    now = get_now() - timedelta(seconds=30)
+    times = list()
+    time_format = "%Y-%m-%dT%H:%M:%SZ"
+    if isinstance(last_fetch, str):
+        times.append(last_fetch)
+        last_fetch = datetime.strptime(last_fetch, time_format)
```

Comment:
dont override function argument `last_fetch` , it is a bad practice. instead define new variable